src/components/ConferencePage/ConferencePage.js

```diff
@@ -63,8 +63,22 @@ export default class ConferencePage extends Component {
 
     this.setState({loading: true, conferences: [], lastLinkFetched: conferenceURL});
 
-    this.fetchConfs(conferenceURL);
-    this.fetchConfs(conferenceURLNextYear);
+    Promise.all([
+      this.fetchConfs(conferenceURL),
+      this.fetchConfs(conferenceURLNextYear),
+    // eslint-disable-next-line promise/always-return
+    ]).then((results) => {
+      const _conferences = results.reduce((concat, current) => {
+        return [...concat, ...current];
+      }, []);
+      const {conferences} = this.state;
+      this.setState({
+        loading: false,
+        conferences: [...conferences, ..._conferences],
```

Comment:
At this point we want to reset all the `conferences` so I would just do:
```
this.setState({
  loading: false,
  conferences
});
```

Reply:
Right. Good catch!

Reply:
Love the `Promise.all` by the way! :)

Reply:
First time I got to use it for real 😄 
I just pushed that last change. Thank you for your patience with the review!

Reply:
I have to say, it's a good place to use it 👌🏼
No prob for the review, thanks for taking the time to add this feature!